scripts/qa_crowbarsetup.sh

```diff
@@ -5153,6 +5153,9 @@ zypper --non-interactive --gpg-auto-import-keys --no-gpg-checks install ses-upgr
         ssh $node "reboot"
         wait_for 100 3 "! nc -w 1 -z $node 22" "node $node to go down"
         wait_for 200 3 "nc -w 1 -z $node 22" "node $node to be back online"
+        # wait for HEALTH_OK or HEALTH_WARN status (WARN might be there because of require_jewel_osds not set yet)
+        wait_for 50 5 'health=$(ssh $node ceph health); echo "$health" | grep -E "HEALTH_(OK|WARN)"' \
```

Comment:
two reasons:
1. I didn't know it exists (reason invalid now)
2. it is not used in the surrounding code